server/src/main/java/com/genymobile/scrcpy/Settings.java

```diff
@@ -2,7 +2,7 @@
 
 import com.genymobile.scrcpy.wrappers.ContentProvider;
 import com.genymobile.scrcpy.wrappers.ServiceManager;
-
+import android.system.Os;
```

Comment:
Perhaps preserve the empty line that was here (and I think this import should go after line 6, to preserve alphabetical order; your IDE should do this perhaps)

Reply:
Added the newline and reordered the import statements. Thank you for the help.